presto-hive/src/main/java/io/prestosql/plugin/hive/util/Statistics.java

```diff
@@ -453,9 +454,13 @@ private static OptionalDouble getDoubleValue(ConnectorSession session, Type type
         return block.isNull(0) ? Optional.empty() : Optional.of(LocalDate.ofEpochDay(((SqlDate) type.getObjectValue(session, block, 0)).getDays()));
     }
 
-    private static OptionalLong getTimestampValue(Block block)
+    private static OptionalLong getTimestampValue(ConnectorSession session, Type type, Block block)
     {
-        return block.isNull(0) ? OptionalLong.empty() : OptionalLong.of(floorDiv(block.getLong(0, 0), MICROSECONDS_PER_SECOND));
+        if (block.isNull(0)) {
+            return OptionalLong.empty();
+        }
+        long epochMicros = ((SqlTimestamp) type.getObjectValue(session, block, 0)).getEpochMicros();
+        return OptionalLong.of(floorDiv(epochMicros, MICROSECONDS_PER_SECOND));
```

Comment:
Here we truncate picos-of-micro, and then truncate micros-of-second.
This is correct for "min" value, but not for the "max"

Anyway, since the method is unused, please just remove it, not fix.